### ISSUE TITLE
[BUG]:@azure/microsoft-playwright-testing package gives error with Playwright OSS 1.49

### ISSUE THREAD

**Issue by vvs11** [microsoft/playwright-testing-service#151]
If you are using @azure/microsoft-playwright-testing-testing package with Playwright OSS1.49, you will see this error:

```
Argument of type 'PlaywrightTestConfig<{}, {}>' is not assignable to parameter of type 'PlaywrightConfigInput'.
  Types of property 'globalSetup' are incompatible.
    Type 'string | string[] | undefined' is not assignable to type 'string | undefined'.
      Type 'string[]' is not assignable to type 'string'.ts(2345)
```

Our team is working on fixing it. As a workaround, **please downgrade to Playwright 1.48**

**Comment by miguelofbc** [comment]
Do you have any idea on the ETA for the fix of this issue?

**Comment by Sid200026** [comment]
Hi @miguelofbc, the fix will be available by 28th Nov, Thursday.

**Comment by Sid200026** [comment]
We have published an alpha version with the fix - `1.0.0-alpha.20241126.2` (https://www.npmjs.com/package/@azure/microsoft-playwright-testing/v/1.0.0-alpha.20241126.2)

**Comment by Sid200026** [comment]
We have released 1.0.0-beta.6 with the fix, please reopen this thread if the issue persists.
